libs/components/modals/src/lib/modules/modal/modal.interface.ts

```diff
@@ -45,6 +45,13 @@ export interface SkyModalConfigurationInterface {
    */
   ariaLabelledBy?: string;
 
+  /**
+   * Specifies an ARIA label for the modal. This sets the modal's `aria-label`
+   * attribute [to support accessibility](https://developer.blackbaud.com/skyux/learn/accessibility).
+   * If the `sky-modal-header` component displays a visible title, do not use this property.
```

Comment:
```suggestion
   * If the `sky-modal-header` component displays a visible title, use `ariaLabelledBy` instead.
```
It seems like the recommendation here might be a little bit off. I don't know if there is any harm in using property when there is a visible title, but based on similar components we have properties for the aria label, it seems like we should recommend the `ariaLabelledBy` instead of recommending against `ariaLabel`. Similar to what we do in existing components, such as colorpicker, that have both a "label" and a "labelledBy" property. I'd recommend tweaking this last sentence to follow that pattern unless I'm incorrect in assuming that this works like those other components.

Also, if this change is correct, I'd also recommend adding a similar sentence to the end of the `ariaLabelledBy` description: "If the modal does not include a visible label, use `ariaLabel` instead."